File: python/283.py
class Solution(object):
    def moveZeroes(self, nums):
        """
        :type nums: List[int]
        :rtype: void Do not return anything, modify nums in-place instead.
        """
        nums[:] = [v for v in nums if v] + [0] * nums.count(0)

File: cpp/283.cpp
#include <vector>
using namespace std;

class Solution {
public:
  void moveZeroes(vector<int> &nums) {
    int n = nums.size();
    int i = 0;
    for (int j = 0; j < n; j++)
      if (nums[j])
        nums[i++] = nums[j];
    while (i < n)
      nums[i++] = 0;
  }
};
